tests/testDisplay.py

```diff
@@ -90,6 +90,36 @@ def testWith(self):
         with afwDisplay.getDisplay(0) as disp:
             self.assertIsNotNone(disp)
 
+    def testDefaultBackend(self):
+        """Test setting and getting of default backend"""
+        if (backend != 'virtualDevice'):
```

Comment:
Thank you for adding these tests!

However: when this is run by Jenkins, isn't the `backend` always going to be `"virtualDevice"`? If so, these tests — and particularly `testClearDefaults()`, below — aren't really testing very much.